pkg/operator/ceph/agent/agent.go

```diff
@@ -275,6 +275,7 @@ func (a *Agent) createAgentDaemonSet(namespace, agentImage, serviceAccount strin
 	} else {
 		logger.Infof("rook-ceph-agent daemonset started")
 	}
+	k8sutil.AddRookVersionLabelToDaemonSet(ds)
```

Comment:
Why this is needed? If needed this should be a separate commit

Reply:
> Why this is needed? If needed this should be a separate commit

Resolved.